sdk/identity/identity/src/credentials/clientSecretCredential.browser.ts

```diff
@@ -73,28 +73,28 @@ export class ClientSecretCredential implements TokenCredential {
     );
     try {
       const urlSuffix = getIdentityTokenEndpointSuffix(this.tenantId);
-      const webResource = this.identityClient.createWebResource({
+      const request = createPipelineRequest({
         url: `${this.identityClient.authorityHost}/${this.tenantId}/${urlSuffix}`,
         method: "POST",
-        disableJsonStringifyOnBody: true,
-        deserializationMapper: undefined,
         body: qs.stringify({
```

Comment:
You managed to remove the dependency on `qs` in IdentityClient by using `URLSearchParams#toString` instead of `qs.stringify`. Any reason not to take the same approach here for consistency's sake?

Reply:
Sure! I wasn’t thinking on replacing it everywhere since we’re tracking that on a separate issue, but I can do this here too.